pom.xml

```diff
@@ -618,5 +612,33 @@
                 <module>flink</module>
             </modules>
         </profile>
+        <profile>
+            <id>hadoop3.0</id>
+            <activation>
+                <property>
+                    <name>hadoop</name>
+                    <value>3.0</value>
+                </property>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <hadoop.version>3.2.4</hadoop.version>
+                <guava.version>27.0-jre</guava.version>
+            </properties>
+        </profile>
+        <profile>
+            <id>hadoop2.10</id>
+            <activation>
+                <property>
+                    <name>hadoop</name>
+                    <value>2.10</value>
+                </property>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <hadoop.version>2.10.2</hadoop.version>
+                <guava.version>11.0.2</guava.version>
+            </properties>
+        </profile>
```

Comment:
Do we need profiles for this? It looks like for the CI (except for python), Spark 3 uses Hadoop 3 and Spark 2 uses Hadoop 2, can we just set the hadoop version in those existing profiles?